environment.d.ts

```diff
@@ -1,12 +1,11 @@
 declare global {
-    namespace NodeJS {
-      interface ProcessEnv {
-        NOTION_SECRET: string;
-        NOTION_DATABASE_ID: string;
```

Comment:
Is moved to the constants file.  (If we have 2 notions 1 for testing 1 for production it could be in the ENV, but is not the case, and we require 5 different databases so would be complex)

(Actually we were not using it anywhere, we added at the beginning but as we have the bookingId for the first task was not needed to mention the database ID)
We require to fetch all records OR create new items